game.py

```diff
@@ -20,8 +20,24 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
-
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while True:
+        letter_typed = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if letter_typed in snowman_word:
+            correct_letter_guess_statuses[letter_typed] = True
+            print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+            if is_word_guessed(snowman_word, correct_letter_guess_statuses) is True:
+            # if snowman_word == generate_word_progress_string(snowman_word, correct_letter_guess_statuses):
+                print("Congratulations, you win!")
+                return
```

Comment:
Nice use of the `return` keyword. This allows for us to exit not only the while loop early if this condition is met but also the function as a whole.